src/components/pages/dashboard/dashboard.test.js

```diff
@@ -1,17 +1,29 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React from 'react';
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
```

Comment:
With `shallow` we are not loading the whole component. If we want to write integration test, will we be able to use `mount` and mock out the service layer?

Reply:
With shallow we load the entire component just not it's children. For integration we can use mount. If you look at the [main app test](https://github.com/Azure/pcs-remote-monitoring-webui/blob/buildFixes/src/components/app/app.test.js) it is still using `mount` to test the deep component integration and routing.